processing/src/main/java/org/apache/druid/segment/incremental/IncrementalIndexRowSelector.java

```diff
@@ -101,4 +108,10 @@ public interface IncrementalIndexRowSelector extends ColumnInspector
    * @return          is the value null for this row?
    */
   boolean isNull(int rowOffset, int aggOffset);
+
+  ColumnFormat getColumnFormat(String columnName);
+
+  int size();
```

Comment:
what size is it exactly?

Reply:
number of rows in the facts table, like after rollup if it is a rollup facts table, will add javadoc and maybe rename, i just picked this up since was previous name on `IncrementalIndex`